src/node/rpc/memberfrontend.h

```diff
@@ -400,6 +402,61 @@ namespace ccf
 
         return jsonrpc::success(complete_proposal(args.tx, vote.id));
       };
+
+      auto create = [this](RequestArgs& args) {
```

Comment:
AIUI, this should be moved to `nodefrontend.h`?

Reply:
discussed offline